pkg/virt-controller/watch/vm/vm.go

```diff
@@ -3191,16 +3191,37 @@ func (c *Controller) sync(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineI
 		return vm, common.NewSyncError(fmt.Errorf(fetchingRunStrategyErrFmt, err), FailedCreateReason), err
 	}
 
-	// Ensure we have ControllerRevisions of any instancetype or preferences referenced by the VM
-	err = c.instancetypeMethods.StoreControllerRevisions(vm)
-	if err != nil {
-		log.Log.Object(vm).Infof("Failed to store Instancetype ControllerRevisions for VirtualMachine: %s/%s", vm.Namespace, vm.Name)
-		c.recorder.Eventf(vm, k8score.EventTypeWarning, common.FailedCreateVirtualMachineReason, "Error encountered while storing Instancetype ControllerRevisions: %v", err)
-		return vm, common.NewSyncError(fmt.Errorf("Error encountered while storing Instancetype ControllerRevisions: %v", err), common.FailedCreateVirtualMachineReason), nil
+	// TODO(lyarwood): Extract all instance type logic into a handler
+	switch c.clusterConfig.GetInstancetypeReferencePolicy() {
+	case virtv1.Reference:
+		// Ensure we have ControllerRevisions of any instancetype or preferences referenced by the VM
+		if err = c.instancetypeMethods.StoreControllerRevisions(vm); err != nil {
+			log.Log.Object(vm).Infof("failed to store Instancetype ControllerRevisions for VirtualMachine: %s/%s", vm.Namespace, vm.Name)
+			c.recorder.Eventf(vm, k8score.EventTypeWarning, common.FailedCreateVirtualMachineReason, "Error encountered while storing Instancetype ControllerRevisions: %v", err)
+			return vm, common.NewSyncError(fmt.Errorf("Error encountered while storing Instancetype ControllerRevisions: %v", err), common.FailedCreateVirtualMachineReason), nil
+		}
+	case virtv1.Expand:
+		expandVMCopy := vm.DeepCopy()
+		if err = c.instancetypeMethods.ApplyToVM(expandVMCopy); err != nil {
```

Comment:
```suggestion
		if err := c.instancetypeMethods.ApplyToVM(expandVMCopy); err != nil {
```

Reply:
After applying changes to the VM, do we need to run any of the validations that are currently found in the admission/mutation webhooks?

Reply:
We've already ran the same validations against the expanded VM during submission and these will run again with the update so no I don't think so.